Dockerfile

```diff
@@ -23,12 +23,8 @@ RUN yum install -y --setopt skip_missing_names_on_install=False \
         | { ! grep -q '^No Match for argument '; } \
     && \
     yum clean all && \
-    rpm2cpio clamav-0*.rpm | cpio -idmv && \
-    rpm2cpio clamav-lib*.rpm | cpio -idmv && \
-    rpm2cpio clamav-update*.rpm | cpio -idmv && \
-    rpm2cpio gnutls*.rpm | cpio -idmv && \
-    rpm2cpio json-c*.rpm | cpio -idmv && \
-    rpm2cpio libtasn1*.rpm | cpio -idmv && \
+    { compgen -G "*.rpm" > /dev/null || { echo 'No RPMs downloaded!'; exit 1; }; } && \
+    for rpm in *.rpm; do rpm2cpio $rpm | cpio -idmv; done && \
```

Comment:
Interesting approach to checking for the presence of files matching the pattern :thinking: 

We want to fail the build if _any_ of the rpms fail to be downloaded which is what your `tee | grep` dance does, so that should cover the case where no rpms are downloaded, right?. In which case, this might be simpler with `shopt -s nullglob` which just causes patterns that expand to nothing to be replaced by the empty string, rather than themselves ([docs](https://www.gnu.org/software/bash/manual/bash.html#The-Shopt-Builtin)).